TELCODOCS-1619-414 Document Telco 4.14 late known issues RNs (#67571)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

* For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the GCP internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])

> * Large latency spikes may be seen whenever a container is started, on an {sno-caps} with guaranteed cpus and load balancing disabled. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])

Reviewer: Would be good to specify what load balancing we refer to.
I.e. what is it that is load balanced or not

Author: Good point it is irq-load-balancing.crio.io to be honest there is an open question on whether I can report as noted as Red Hat internal

Author: @djuran changed to _with guaranteed CPUs and Interrupt Request (IRQ) load balancing is disabled_

> * Large latency spikes may be seen whenever a container is started, on an {sno-caps} with guaranteed CPUs and Interrupt Request (IRQ) load balancing is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])

Reviewer 2: ```suggestion
* On an {sno-caps} that has guaranteed CPUs and where Interrupt Request (IRQ) load balancing is disabled, large latency spikes can occur at container startup. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```

Just a thought--attempting to make it easier to catch a note applicable to one's situation while scanning.

Not sure if I restructured the restrictive clause correctly.

> * When deploying an application with a large number of pods, some of which have CPU limits configured, the deployment can sometimes fail. The workaround is to re-deploy the application. (link:https://issues.redhat.com/browse/RHEL-7232[*RHEL-7232*])

Reviewer 2: ```suggestion
* When deploying an application that has a large number of pods, some of which have CPU limits configured, the deployment can fail. The workaround is to re-deploy the application. (link:https://issues.redhat.com/browse/RHEL-7232[*RHEL-7232*])
```

Reviewer 2: ISG asks for "that" w/restrictive clauses.

> * On an {sno-caps} with disabled capabilities, the `openshift-controller-manager-operator` may continuously get restarted.  The workaround is to enable the build capability or to manually create the `builds.config.openshift.io` CRD.

Reviewer 2: ```suggestion
* On an {sno-caps} that has disabled capabilities, the `openshift-controller-manager-operator` may continuously restart.  As a workaround, enable the build capability or  manually create the `builds.config.openshift.io` CRD.
```

Reviewer 2: Will "enable the build capability" make sense to the audience?

+
Perform the following steps to manually create the `builds.config.openshift.io` CRD:
+
--

> . Extract the release manifests:

Reviewer 2: Recall the "run the following command" language required by the repo contribution guidelines.
